### PR TITLE
Feauture register step one

### PR DESCRIPTION
**Pull request by tslekwadi**

### CODE REVIEW

File: src/containers/register/index.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
+
+   .registerForm
+   {	
+       border-radius: 12px;	
+       margin: auto;
+       height: 644px;	
+       width: 533px;	
+       border-radius: 12px;
+    background-color: #01B9BD;
```
Comment (bsibiya) [inline src/containers/register/index.css]: Indentation should be consistent

File: src/containers/register/registerComponent.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                  <input type="text" class="form-field-white" />
+              </label>
+  
+              <label>Password
+                  <input type="text"  />
+              </label>
+          
+              <input class="rectangleButton" type="submit" value="Submit" />
+  
+
```
Comment (bsibiya) [inline src/containers/register/registerComponent.js]: indentation and spaces

File: src/containers/register/index.css (hunk 1 above)
Comment (WesleyBuck) [reply src/containers/register/index.css]: 👍

**Review by WesleyBuck — CHANGES_REQUESTED** [review]
Indentation needs to be correct!

**Review by bsibiya — APPROVED** [review]

File: src/containers/register/index.css (hunk 1 above)
Comment (tslekwadi) [reply src/containers/register/index.css]: Changes updated

File: src/containers/register/index.css [hunk 3]
```diff
@@ -22,36 +21,36 @@
...
-     color: #FFFFFF;	
-     font-family: "Avenir Next";	
-     font-size: 46px;	
-     font-weight: 510;	
-     letter-spacing: 2px;	
-     line-height: 49px;
-     margin-bottom: -5%;
- }
+{	
+    margin-right:-30px;
```
Comment (grootendorste) [inline src/containers/register/index.css]: I can see there's no space here.

File: src/containers/register/index.css [hunk 4]
```diff
@@ -22,36 +21,36 @@
...
+.rectangle-button 
+{	
+    margin-top: 35px !important;
+    height: 57px !important;	
+    width: 197px !important;	
+    border-radius: 5px !important;	
+    border-width:0px !important;
+    background-color: #0DABAF !important;	
+    box-shadow: 0 2px 6px 0 rgba(0,0,0,0.25) !important;
+    margin-left:10%;
```
Comment (grootendorste) [inline src/containers/register/index.css]: Another slight issue.

**Review by grootendorste — CHANGES_REQUESTED** [review]
2 slight changes. Will take a second if you don't mind.

**Review by grootendorste — APPROVED** [review]

**Review by WesleyBuck — APPROVED** [review]

**Pull request merged**